pkg/apis/kops/validation/validation.go

```diff
@@ -829,6 +829,9 @@ func validateKubeAPIServer(v *kops.KubeAPIServerConfig, c *kops.Cluster, fldPath
 		}
 	}
 
+	if c.IsKubernetesLT("1.22") && len(v.AdditionalServiceAccountIssuers) > 0 {
+		allErrs = append(allErrs, field.Forbidden(fldPath.Child("additionalServiceAccountIssuers"), "additionalServiceAccountIssuers requires Kubernetes 1.22+"))
+	}
```

Comment:
No need for this:
https://github.com/kubernetes/kops/blob/c6bb40ed328c99ca1490113334f61622600c5155/upup/pkg/fi/cloudup/apply_cluster.go#L73